marlowe-playground-client/webpack.config.js

```diff
@@ -40,22 +46,10 @@ module.exports = {
     entry: "./entry.js",
     output: {
         path: path.join(__dirname, "dist"),
-        filename: "[name].[hash].js",
+        filename: "[name].[fullhash].js",
         pathinfo: true,
         clean: true,
     },
-    optimization: {
```

Comment:
The separation was on the "bundles split" level but not on the build level. `optimization` section configures the behavior of `CommonChunksPlugin` and it **doesn't** prevent rebuilds of vendor bundle on every change. In general this optimization is worth doing when you have multiple applications separate entry points and you want to share a common bundle between them.
This new, `DllPlugin` based approach also provides a separation of bundles but also "freezes" the vendor bundle. Invalidation of this vendor cache can be sometimes problematic ([we can be sure about that :-P](https://www.google.com/search?q=There+are+only+two+hard+things+in+Computer+Science%3A+cache+invalidation+and+naming+things.&oq=There+are+only+two+hard+things+in+Computer+Science%3A+cache+invalidation+and+naming+things.&aqs=chrome.0.69i59.1165j0j7&sourceid=chrome&ie=UTF-8)) but should speed up devel cycle because there are at least three peformance-wise benefits of precached js: less js to bundle, less size of the main bundle to reload on every rebuild, and more js code already parsed and ready to use on the browser side. In this context I propose also caching the largest PS libs (this is not included in this PR, but I'm happy to add it).
Also the difference between these two plugins as I've pointed out above could be found here: https://stackoverflow.com/a/43876021